wrappers/java/src/test/java/org/hyperladger/indy/sdk/pairwise/CreatePairwiseTest.java

```diff
@@ -0,0 +1,72 @@
+package org.hyperladger.indy.sdk.pairwise;
+
+import org.hyperledger.indy.sdk.ErrorCode;
+import org.hyperledger.indy.sdk.ErrorCodeMatcher;
+import org.hyperledger.indy.sdk.IndyIntegrationTest;
+import org.hyperledger.indy.sdk.pairwise.Pairwise;
+import org.hyperledger.indy.sdk.signus.Signus;
+import org.hyperledger.indy.sdk.signus.SignusResults.CreateAndStoreMyDidResult;
+import org.hyperledger.indy.sdk.wallet.Wallet;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.ExecutionException;
+
+public class CreatePairwiseTest extends IndyIntegrationTest {
+
+	private Wallet wallet;
+	private String walletName = "pairwiseWallet";
+	private String unknownDid = "NcYxiDXkpYi6ov5FcYDi1e";
+	private String myDid;
+	private String theirDid;
+
+	@Before
+	public void createWallet() throws Exception {
```

Comment:
There are a lot of same `@Before` and `@After` with creating simple wallet. So I suggest to create `class IndyIntegrationTestWithSingleWallet extends IndyIntegrationTest`

Reply:
Not sure that this PR is the right place for this refactoring. I suggest to merge this and add some technical debt task.